Resources/Private/BackendModule/src/Components/Root.tsx

```diff
@@ -1,29 +1,34 @@
 import React from "react";
 import PropTypes from "prop-types";
-import AssetList from "./AssetList";
+import ItemsList from "./ItemsList";
 import {Provider} from "react-redux";
-import ConfigurationForm from "./ConfigurationForm";
 import StartModuleButton from "./StartModuleButton";
 import SubmitAndFetchNextButton from "./SubmitAndFetchNextButton";
 import PureComponent from "./PureComponent";
 import ErrorMessage from "./ErrorMessage";
+import ConfigurationForm from "./ConfigurationForm";
+import ReturnToOverviewButton from "./ReturnToOverviewButton";
+import {Store} from "redux";
+import {Endpoints} from "../Model/Endpoints";
 
 export default class Root extends PureComponent {
     static propTypes = {
-        store: PropTypes.object.isRequired
+        store: PropTypes.object.isRequired,
+        endpoints: PropTypes.object.isRequired
     }
 
     render() {
-        const {store} = this.props;
+        const {store, endpoints}: {store: Store, endpoints: Endpoints} = this.props;
         return (
             <Provider store={store}>
                 <div className={'neos-content neos-indented neos-fluid-container'}>
                     <ErrorMessage />
```

Comment:
Discussion:
Ich würde erwarten dass einen Componente ErrorMessage eine Prop Message hat und die dann rendert, aktuell unterscheidest du nicht zwischen Components die nur von außen gesteurt werden, und denen die innere Logik haben.

Ich fände es logischer den State in der App zu steurn und hier folgendes schreiben:
```{harError ? <ErrorMessage message={errorMessage} /> : ''}```